modules/cart/src/CartManager.php

```diff
@@ -135,6 +136,13 @@ public function updateLineItem(OrderInterface $cart, LineItemInterface $line_ite
     $line_item->save();
     $event = new CartLineItemUpdateEvent($cart, $line_item, $original_line_item);
     $this->eventDispatcher->dispatch(CartEvents::CART_LINE_ITEM_UPDATE, $event);
+    // Let's *always* update the order if a line item is changed.
+    //
+    // The only value you can change by default is the amount via the cart view.
+    // Once this changes, a cart update is required for the total_price__amount
+    // db column. We do not, like the addLineItem() function, test for
+    // quantities and a possible difference, we expect we need to update it.
```

Comment:
agreed, but figured: now it's clear for the review. I'll update it to "We need to recalculate the order's total".